tests/end_to_end/candid_rpc/functional_syntax/motoko_examples/http_counter/src/index.ts

```diff
@@ -185,7 +185,7 @@ export default Canister({
                     };
                 }
                 default: {
-                    return ic.trap('unreachable');
+                    return ic.trap('should not happen');
```

Comment:
you never did like the word unreachable did you? Is that all that's going on here?